<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WellsBrowserForm</class>
 <widget class="QWidget" name="WellsBrowserForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="mWellsFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QTreeView" name="mWellsTreeView">
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="headerStretchLastSection">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="mWellLabel">
          <property name="text">
           <string>Скважины</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mToggleSelected">
          <property name="toolTip">
           <string>Переключить выделение подсвеченных строк</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mSelectAll">
          <property name="toolTip">
           <string>Выделить все</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mUnselectAll">
          <property name="toolTip">
           <string>Снять выделение</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mSaveWellList">
          <property name="toolTip">
           <string>Создать список скважин из отмеченных</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mWellListToolButton">
          <property name="toolTip">
           <string>Список скважин</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mWellFilterToolButton">
          <property name="toolTip">
           <string>Фильтровать список</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionSetupFilter">
   <property name="text">
    <string>Установка...</string>
   </property>
   <property name="toolTip">
    <string>Настроить фильтр</string>
   </property>
  </action>
  <action name="actionSelectList">
   <property name="text">
    <string>Выбрать список...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
